docs_nnx/conf.py

```diff
@@ -113,7 +113,7 @@
 # href with no underline and white bold text color
 announcement = """
 <a
-  href="https://flax-linen.readthedocs.io/en/latest"
+  href="https://flax.readthedocs.io/en/latest"
```

Comment:
We need to link to `flax-linen` here.

Reply:
Got it @cgarciae 

Local builds seem fine even before the changes in this PR. ReadTheDocs' builds have the strange redirect for Glossary. As @IvyZX suggested maybe it's the redirect in the RTD config.

As discussed, will also rename the file to `nnx_glossary.rst` and see if that makes a difference.